src/tools/continuousintegration.rst

```diff
@@ -79,6 +79,101 @@ the button “Run pipeline” on a specific branch (i.e. master).
 
 |image5|
 
+Collecting & consolidating CI health metrics as part of the CI pipeline
+-----------------------------------------------------------------------
+As part of the CI/CD process all teams are expected to collect and consolidate
+the necessary code health metrics under a file named :code:`ci-metrics.json` at
+the root of the repositories they have ownership over.
```

Comment:
When you say _at the root of the repository_ it gives the impression that the file has to be placed inside the repository as part of the git filesystem. 
I would add something explaining where the file must be placed more clearly.

Reply:
There was already a paragraph further down explaining this, but I also altered the paragraph you mention to make it clearer.